addons/skin.estuary/xml/View_503_NowPlaying.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<includes>
+	<include name="View_503_NowPlaying">
+		<control type="group">
+			<include>OpenClose_Right</include>
+			<left>582</left>
```

Comment:
it's not properly aligned...the list is overlapping with the info panel on the left

![align](https://user-images.githubusercontent.com/687265/70397727-f462b880-1a14-11ea-800a-91daf91e5a3d.jpg)